packages/flutter/test/material/switch_list_tile_test.dart

```diff
@@ -298,4 +298,44 @@ void main() {
     await tester.pump();
     expect(Focus.of(childKey.currentContext, nullOk: true).hasPrimaryFocus, isFalse);
   });
+  
```

Comment:
```suggestion

```

Reply:
But, isn't that a new line? I mean, in previous PRs it never gave this error.

Reply:
It's not the new line. It's the whitespace within the line. 
```
{whitespace}{whitespace}
```
vs
```

```

Reply:
I see, thanks for clearing silly doubt @shihaohong.
I'll do the required changes.